src/main/java/bio/terra/janitor/service/janitor/JanitorApiService.java

```diff
@@ -0,0 +1,73 @@
+package bio.terra.janitor.service.janitor;
+
+import bio.terra.generated.model.*;
+import bio.terra.janitor.common.exception.InternalServerErrorException;
+import bio.terra.janitor.db.*;
+import bio.terra.janitor.service.iam.AuthenticatedUserRequest;
+import bio.terra.janitor.service.iam.IamService;
+import java.util.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.support.TransactionTemplate;
+
+@Component
+public class JanitorApiService {
```

Comment:
add a comment about what this service is for

Reply:
Done